adserver/templates/adserver/reports/advertiser.html

```diff
@@ -19,32 +19,38 @@
 <section class="mb-5">
   {% if flights %}
     <h2>{% trans 'Flight report details' %}</h2>
-    <div class="table-responsive">
-      <table class="table table-hover">
-        <thead>
-          <tr>
-            <th><strong>{% trans 'Flight Name' %}</strong></th>
-            <th><strong>{% trans 'Start date' %}</strong></th>
-            <th><strong>{% trans 'End date' %}</strong></th>
-          </tr>
-        </thead>
-        <tbody>
-          {% for flight in flights %}
-            <tr>
-              <td>
-                <a href="{% url 'flight_report' advertiser.slug flight.slug %}?start_date={{ start_date|date:'Y-m-d' }}{% if end_date %}&amp;end_date={{ end_date|date:'Y-m-d' }}{% endif %}">{{ flight.name }}</a>
-                {% if not flight.live %}
-                  <span class="fa fa-eye-slash fa-fw text-muted" aria-hidden="true" data-toggle="tooltip" title="{% trans 'This flight is disabled' %}"></span>
-                {% endif %}
-              </td>
-              <td>{{ flight.start_date }}</td>
-              <td>{{ flight.end_date }}</td>
-            </tr>
-          {% endfor %}
-        </tbody>
-      </table>
-    </div>
-  {% endif %}
+
+    {% regroup flights by state as flight_groups %}
+    {% for state, flight_list in flight_groups %}
+      {% if flight_list %}
+        <div class="table-responsive">
+          <table class="table table-hover">
+            <thead>
+              <tr>
+                <th><strong>{% blocktrans %}{{ state }} flights{% endblocktrans %}</strong></th>
```

Comment:
Are we losing the flight name display, or is it shown somewhere else?

Reply:
I'm not sure exactly what you mean by that, but the screenshot should illustrate what it will look like. There won't be a table heading "Flight name" but will instead say "Current flights", "Past flights", etc.

Reply:
Ah yea, I see how it works now. Makes sense 👍